pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/httproute_reconciler.go

```diff
@@ -657,38 +657,57 @@ func (r *RawHTTPRouteReconciler) reconcileExplainerHTTPRoute(ctx context.Context
 	if err != nil {
 		return err
 	}
+
+	// reconcile httproute
```

Comment:
nit: this block is re-used multiple times. I'd prefer if it was extracted into a function

Reply:
I think this will be covered by the follow up to refactor these to use CheckResultType anyway